app/controllers/internal/configs_controller.rb

```diff
@@ -73,6 +74,18 @@ def clean_up_params
     config[:suggested_tags] = config[:suggested_tags].downcase.delete(" ") if config[:suggested_tags]
     config[:authentication_providers] = config[:authentication_providers].downcase.delete(" ") if config[:authentication_providers]
     config[:sidebar_tags] = config[:sidebar_tags].downcase.delete(" ") if config[:sidebar_tags]
+
+    ensure_proper_date(config)
+  end
+
+  def ensure_proper_date(config)
+    return unless config[:campaign_stories_start_date]
+
+    begin
+      Date.parse(config[:campaign_stories_start_date])
```

Comment:
This will parse anything resembling a date, with possible unpredictable results:

```ruby
[1] pry(main)> Date.parse("May I have a cup of coffee, please?")
=> Fri, 01 May 2020
[2] pry(main)> Date.parse("Whatever in September")
=> Tue, 01 Sep 2020
```

I would also like to propose using a ISO date, as `04/03/2020` for a European is the 4th day of March, but for Americans is the 3rd day of April and this is bound to create confusion :-)

We can use `Date.iso8601`:

```ruby
[4] pry(main)> Date.iso8601("Whatever in September")
Date::Error: invalid date
from (pry):4:in `iso8601'
[5] pry(main)> Date.iso8601("20200403")
=> Fri, 03 Apr 2020
[6] pry(main)> Date.iso8601("2020-04-03")
=> Fri, 03 Apr 2020
```